devito/archinfo.py

```diff
@@ -151,6 +151,7 @@ class Platform(object):
 
     def __init__(self, name, **kwargs):
         self.name = name
+        self.known_isas = []
```

Comment:
this should be a class attribute, not an instance attribute. And it can even be a tuple, instead of a list, since the ISAs are fixed on each architecture.